pages/phases/job-analysis/creating-assessments-from-competencies.md

```diff
@@ -37,18 +39,54 @@ If you are creating a new passing score work simulation assessment, think throug
+
+#### Work sample essay example:
+Imagine you are asked to modernize and update [example]. Create a research plan to outline how you would identify pain points for [example].
+
+### General essay: short answer
+This type of assessment requires the applicant to provide short answer responses to a set of questions based on a given scenario. Each question should be tied to a specific competency and have explicit scoring criteria and instructions. You may have one or multiple questions per competency. This method is more open-ended and can be subject to a SME’s interpretation. 
+
+#### Short answer example
+There are currently 10 different systems that support 30 different programs within an agency. The agency plans to consolidate these 10 systems into just one. You have been brought on to help with this initiative.
+
+1.	What questions would you want to answer? What assumptions would you make?
+2.	What would you propose to the agency for how to accomplish this?
+3.	What would a potential project plan / roadmap look like?
+
+### General essay: long answer
+This type of assessment requires the applicant to provide a longer essay response based on a given scenario. The essay response should allow applicants to demonstrate their proficiency level across the various competencies. This method is more open-ended and can be subject to a SME’s interpretation.
+
+#### Long answer prompt example
+You’ve been tasked with solving “x” using “y” methods. Detail a plan of action, with the steps you’d take. In your plan, account for various risks, such as what you would do if a stakeholder blocked your efforts and alternative methods you would consider.
+
+### Written assessments tips
 - The assessment type should be appropriate to evaluate the applicant's proficiency level within the pre-defined competencies.
 - Ideally, you will have time during the workshop for live testing of assessment questions. If you end up revising the assessment after the workshop, the revised assessment will need to be tested.
 - As part of testing, record the time it will take to conduct the written assessment. Add that to the job announcement and assessment instructions.
 - You should provide an estimated timeframe within the JOA in which applicants can expect a written assessment to be sent (e.g., "If you are selected to move forward, you should expect to receive a written assessment within approximately 3-5 days of this announcement closing."
 - To keep the applicant burden manageable, give applicants a maximum word count to use to complete the assessment.
 
-### If Using Two Live Interviews
+## Structured Interviews
+
+All pilots to date have included a structured interview as the final assessment step. We recommend having interviews be the last assessment step because of the time required to schedule, conduct, and assess them.
```

Comment:
```suggestion
All pilots to date have included a structured interview as the final assessment step. The benefit of including a pass/fail interview is that until a SME speaks with an applicant about the technical competencies, it is hard to know if they are really qualified. In the competitive process, you want to make sure only qualified applicants make it through the process to avoid a certificate cancelled with no selections.
```